facility-locator/map.js

```diff
@@ -13,7 +13,9 @@ FacilityLocator = (function() {
       directionsStart,
       geolocated = false; // Used when determining starting location for driving directions
       zoomLevelToShowMarkers = 5;
-      infowindows = {};
+      infowindows = {},
+      // Checks whether the matchMedia is a function (supported) and whether this is a small screen
+      isSmallScreen = !(typeof matchMedia === 'function') && (matchMedia('(max-width: 40.063em)').matches);
```

Comment:
total curiosity and not blocking merging, but @webinista how did you come up with 40.063?